container/service_options.go

```diff
@@ -97,11 +97,15 @@ func (options *ServiceOptions) swarmMounts(force bool) []mount.Mount {
 	return mounts
 }
 
+// swarmNetworks creates all necessary network attachment configurations
+// Each network will be attached based on their networkID but also based on aliases
+// These aliases will make services accessible from other containers inside the same network
 func (options *ServiceOptions) swarmNetworks() (networks []swarm.NetworkAttachmentConfig) {
 	networks = make([]swarm.NetworkAttachmentConfig, len(options.NetworksID))
 	for i, networkID := range options.NetworksID {
 		networks[i] = swarm.NetworkAttachmentConfig{
-			Target: networkID,
+			Target:  networkID,
+			Aliases: options.Namespace,
```

Comment:
Using directly `options.Namespace` will use each string of the namespace as a aliases. Eg:
```go
options.Namespace := []string{"service-hash", "dependency-name", "etc..."}
```
will result in creating multiple aliases:
"service-hash",
"dependency-name",
"etc..."

The alias should be ONLY the name of the dependency defined in the mesg.yml file.
Eg from Anthony mesg.yml used in the first post of this PR: "depA" for the first dependency. And "depB" for the second dependency.

This way, the dependencies' servers are accessible at `depA:1234` and `depB:5678`.